.env.example

```diff
@@ -1,7 +1,8 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-PORT=
-
-DB_TEST_NAME=barefoot_test
+DB_USERNAME=
+DB_PASSWORD=
+DB_NAME=
+DB_TEST_NAME=
+DB_HOST=
+NODE_ENV=test
+SALT=
+SECRET_KEY=
```

Comment:
You can use JWT_SECRET to make it more descriptive

Reply:
ok